### PR TITLE
Ops: increase php memory limit in cli queue worker

### PR DESCRIPTION
**Pull request by indraprasetya154**
## Overview
-Since laravel limit their memory consume for running cli queue:work onl7 128mb, this change will be implement memory usage for queue with 1024mb memory

## Evidence
title: Ops: increase php memory limit in cli queue worker
project: SIDEBAR
participants: @indraprasetya154  @anandarh @azophy

### CODE REVIEW

**Review by azophy — APPROVED** [review]
gaaaassss

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ if [ "$app" = "app" ]; then
 elif [ "$app" = "queue" ]; then
 
     echo "Running the queue..."
-    php artisan queue:work --queue=default --sleep=3 --tries=3
+    php artisan queue:work --queue=default --sleep=3 --tries=3 --memory=1024
```
Comment (azophy) [inline docker/docker-entrypoint.sh:15]: opsi kalau pakai env var mungkin gini:

```
worker_memory_limit=${WORKER_MEMORY_LIMIT:-128}

# .....

php artisan queue:work --queue=default --sleep=3 --tries=3 --memory=$worker_memory_limit
```

File: docker/docker-entrypoint.sh [hunk 2]
```diff
@@ -12,7 +13,7 @@ if [ "$app" = "app" ]; then
 elif [ "$app" = "queue" ]; then
 
     echo "Running the queue..."
-    php artisan queue:work --queue=default --sleep=3 --tries=3
+    php artisan queue:work --queue=default --sleep=3 --tries=3 --memory=queueMemory
```
Comment (azophy) [inline docker/docker-entrypoint.sh:16]: pake `$` ndraa

```suggestion
    php artisan queue:work --queue=default --sleep=3 --tries=3 --memory=$queueMemory
```

Comment (indraprasetya154) [reply docker/docker-entrypoint.sh]: sorry sorry typo typo ketinggalan 🙏

**Review by azophy — APPROVED** [review]
LGTM

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (indraprasetya154) [reply docker/docker-entrypoint.sh]: yesss aku buat variable aja, sudah di update di commit terbaru yak

**Review by anandarh — APPROVED** [review]

**Pull request merged**